server/routes/message.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const { getAllMessages, sendMessage } = require("../controllers/message");
+const { validateConversationId, validateMessageData } = require("../validate");
+
+router.route("/:convoId").get(protect, validateConversationId, getAllMessages);
+router.route("/send").post(protect, validateMessageData, sendMessage);
```

Comment:
I think it might make more sense to relocate these routes to `GET/POST conversations/:id/messages/`

Reply:
Alright, I'll create a new route and controller to handle the creation of a conversation separately. Will be committing the new changes soon :smile: